app-shells/atuin/atuin-18.0.1.ebuild

```diff
@@ -498,8 +509,6 @@ src_install() {
 
 	if use server; then
 		systemd_dounit "${FILESDIR}/atuin.service"
-	else
-		rm -r "docs/docs/self-hosting" || die
```

Comment:
Why is this dropped?

Reply:
As I tried to explain in the PR description, upstream [removed the docs](https://github.com/atuinsh/atuin/pull/1581) from the repository. If you want to keep the docs we would have to source the separate docs repository.

Reply:
I was looking mostly a the commit message, and did not notice it in the github description. The commit message would be the preferred place for this anyways. So in the future, please add such things to the commit message.

Reply:
Meh, it's a pity that they moved the docs into an extra [repo](https://github.com/atuinsh/docs). That makes it only harder for us to ship the documentation as part of the Gentoo package.